pkg/services/accesscontrol/api/api.go

```diff
@@ -25,17 +25,21 @@ type AccessControlAPI struct {
 func (api *AccessControlAPI) RegisterAPIEndpoints() {
 	// Users
 	api.RouteRegister.Get("/api/access-control/user/permissions",
-		middleware.ReqSignedIn, routing.Wrap(api.getUsersPermissions))
+		middleware.ReqSignedIn, routing.Wrap(api.getUserPermissions))
 }
 
 // GET /api/access-control/user/permissions
-func (api *AccessControlAPI) getUsersPermissions(c *models.ReqContext) response.Response {
+func (api *AccessControlAPI) getUserPermissions(c *models.ReqContext) response.Response {
 	reloadCache := c.QueryBool("reloadcache")
 	permissions, err := api.Service.GetUserPermissions(c.Req.Context(),
 		c.SignedInUser, ac.Options{ReloadCache: reloadCache})
 	if err != nil {
 		response.JSON(http.StatusInternalServerError, err)
 	}
 
+	if c.QueryBool("scoped") {
+		return response.JSON(http.StatusOK, ac.GroupScopesByAction(permissions))
```

Comment:
Should this be another endpoint. I don't think a query string should change the type of response, map[string]bool vs map[string][]string

Reply:
It doesn't seem too far fetched, to me, that you get a slightly different response when requesting the scopes on top of the actions. That being said I'm not against having another endpoint. As we discussed, I will however move the current endpoint to:
`GET /api/access-control/user/actions` rather than placing this new code under another endpoint (which I wouldn't know how to name differently than `GET /api/access-control/user/permissions`).
The current `/api/access-control/user/permissions` being undocumented, I think it's ok to change it :slightly_smiling_face: